atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -399,10 +401,11 @@ private void deleteAllTimestampsOverflow(String overflowTable, List<Object[]> ar
         conns.get()
                 .updateManyUnregisteredQuery(
                         " /* DELETE_ALL_TS_OVERFLOW (" + overflowTable + ") */ "
-                                + " DELETE /*+ INDEX(m " + PrimaryKeyConstraintNames.get(overflowTable) + ") */ "
-                                + "   FROM " + overflowTable + " m "
-                                + "  WHERE m.id IN (SELECT /*+ INDEX(i " + PrimaryKeyConstraintNames.get(shortTableName)
-                                + ") */ "
+                                + " DELETE /*+ INDEX(m " + PrimaryKeyConstraintNames.get(overflowTable) + ")"
+                                + "            NO_INDEX_FFS(m " + PrimaryKeyConstraintNames.get(overflowTable) + ") */"
+                                + " FROM " + overflowTable + " m "
+                                + " WHERE m.id IN (SELECT /*+ INDEX(i " + PrimaryKeyConstraintNames.get(shortTableName)
```

Comment:
should we be using some other hint to make sure it executes this inner query first? Do we want a USE_NL and LEADING hints for this?

Reply:
In other words I think we probably actually want this inner `SELECT ...` query to be the outer for loop in the query plan.

Reply:
You are correct. After looking into this more, we want to add the hints `leading(i) use_nl(m)`